README.md

```diff
@@ -18,12 +18,9 @@
-	<a href="https://github.com/catppuccin/userstyles/issues">
-		<img alt="Issues" src="https://img.shields.io/github/issues/catppuccin/userstyles?style=for-the-badge&logo=gitbook&color=B5E8E0&logoColor=D9E0EE&labelColor=302D41"></a>
+	<a href="https://github.com/catppuccin/userstyles/stargazers"><img src="https://img.shields.io/github/stars/catppuccin/userstyles?colorA=363a4f&colorB=b7bdf8&style=for-the-badge"></a>
+  <a href="https://github.com/catppuccin/userstyles/blob/main/styles"><img src="https://img.shields.io/github/directory-file-count/catppuccin/userstyles/styles?label=Userstyles&colorA=363a4f&colorB=cba6f7&style=for-the-badge"></a>
+	<a href="https://github.com/catppuccin/userstyles/contributors"><img src="https://img.shields.io/badge/dynamic/yaml?url=https%3A%2F%2Fraw.githubusercontent.com%2Fcatppuccin%2Fuserstyles%2Fmain%2Fscripts%2Fuserstyles.yml&query=%24.collaborators.length&label=Maintainers&colorA=363a4f&colorB=a6da95&style=for-the-badge"></a>
```

Comment:
```suggestion
	<a href="https://github.com/catppuccin/userstyles/blob/main/.github/CODEOWNERS"><img src="https://img.shields.io/badge/dynamic/yaml?url=https%3A%2F%2Fraw.githubusercontent.com%2Fcatppuccin%2Fuserstyles%2Fmain%2Fscripts%2Fuserstyles.yml&query=%24.collaborators.length&label=Maintainers&colorA=363a4f&colorB=a6da95&style=for-the-badge"></a>
```
Would it be dumb to link to the CODEOWNERS file here? The contributors page works but isn't fully accurate since this badge is just counting maintainers.

Reply:
I think this is a good change